src/Forex/ForexData.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+using Azure.Migrate.Export.Common;
+using Azure.Migrate.Export.HttpRequestHelper;
+using Azure.Migrate.Export.Models;
+
+namespace Azure.Migrate.Export.Forex
+{
+    public static class ForexData
+    {
+        private static Dictionary<string, double> ExchangeRatesUSD;
+        private static double ExchangeRate = 1.0;
+        private static ExchangeRateStates ExchangeRateState = ExchangeRateStates.Unfetched;
+
+        public static Dictionary<string, double> GetExchangeRatesUSD()
+        {
+            return ExchangeRatesUSD;
+        }
+
+        public static ExchangeRateStates GetExchangeRateState()
+        {
+            return ExchangeRateState;
+        }
+
+        public static double GetExchangeRate()
+        {
+            return ExchangeRate;
+        }
+
+        public static void UpdateExchangeRatesWithUSDFallback()

```

Comment:
Do we have exchange rates for all currencies that we support in AME? If yes, then, we don't need a USD fallback.

Reply:
Thanks for the great suggestion, removed USD fallback